### DISCUSSION TITLE
How to configure SDKMan to make releases default?

### DISCUSSION THREAD

**Discussion by martypitt** [jreleaser/jreleaser#875] [Q&A] [answered]
Hi

We use JReleaser on Taxi to push to SDKMan. 

Relevant config files:
 * [jreleaser.yml](https://gitlab.com/taxi-lang/taxi-lang/-/blob/develop/taxi-cli/jreleaser.yml)
 * [pom.xml](https://gitlab.com/taxi-lang/taxi-lang/-/blob/develop/taxi-cli/pom.xml)
 * [gitlabi-ci.yml](https://gitlab.com/taxi-lang/taxi-lang/-/blob/develop/.gitlab-ci.yml#L148)

This all works great, and our releases are pushed to SDKMan.

However, they're not set as the default version of the distribution.  Is this possible?  Am I missing some config?

Thanks for your awesome work here!

**Comment by aalmiray** [comment, marked as answer]
Yes, it's possible. You currently have the value of `command` set to `MINOR`. If you set it to `MAJOR` it will also tell Sdkman to set said version as default. Note, that the project version does not necessarily have to have a bump in its major version number in order to use `MAJOR` as value.

In other words, the only difference between `MINOR` and `MAJOR` is that the later sets the candidate as default as well.

**Reply by martypitt** [reply]
Ah, awesome, thanks!
